internal/pkg/template/templates/environment/cf.yml

```diff
@@ -634,7 +634,7 @@ Outputs:
       Name: !Sub ${AWS::StackName}-SubDomain
 {{- end}}
   EnabledFeatures:
-    Value: !Sub '${ALBWorkloads},${InternalALBWorkloads},${EFSWorkloads},${NATWorkloads}'
+    Value: !Sub '${ALBWorkloads},${InternalALBWorkloads},${EFSWorkloads},${NATWorkloads},${Aliases}'
```

Comment:
A long time ago, we had a chat w @iamhopaul123 because I was concerned that adding `Aliases` to Outputs will cause us to quickly hit the custom resource response limit of 4,096 bytes: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/crpg-ref-responses.html for the `EnvControllerAction`.
This is because the `EnvControllerAction`'s response contains all the Outputs of the env stack.

This has happened in the past: https://github.com/aws/copilot-cli/issues/3298 where the client deployed over 100 services in an environment.

Can I ask two action items please 🥺:

1. Can we validate if an output value can be longer than 4096 bytes? To ensure that v large environments can continue being deployed.
2. Can we change the code for the  `EnvControllerAction` to ignore the `EnabledFeatures` output from the stack? This should make it v likely that we won't hit the response output limit for custom resources.